migrations/20220922230639_insert_default_sync_fn.up.sql

```diff
@@ -0,0 +1,16 @@
+BEGIN;
+CREATE OR REPLACE FUNCTION mergestat.insert_default_sync(imported_repo_id_param uuid, sync_type_param text)
+RETURNS boolean
+AS
+$$
+begin
+IF EXISTS (SELECT sync_type FROM mergestat.repo_syncs WHERE repo_id = imported_repo_id_param AND sync_type=sync_type_param) THEN
+    RETURN TRUE;
+ELSE 
+    INSERT INTO mergestat.repo_syncs (repo_id, sync_type) VALUES(imported_repo_id_param,sync_type_param);
+    RETURN FALSE;
+END IF;
+end;
+$$ LANGUAGE plpgsql;
```

Comment:
rather than defining a DB function for this, I think we ought to do it directly as a statement that `sqlc` uses (see above comment). I think it's straight forward enough to handle that way, and then we don't need a migration for this functionality

Reply:
Yeak  makes sense , I was making this change because of the enqueue misconception I had ;/